src/seedu/addressbook/AddressBook.java

```diff
@@ -1150,10 +1154,9 @@ private static String getUsageInfoForExitCommand() {
      * @param sign  Parameter sign to be removed
      * @return  string without the sign
      */
-    private static String removePrefixSign(String s, String sign) {
-        return s.replace(sign, "");
+    private static String removePrefix(String string, String prefix) {
+        return string.replaceFirst(prefix, "");
```

Comment:
You need to specifically only replace prefix if it occurs at the start of fullString.